package.json

```diff
@@ -57,7 +57,12 @@
     "start": "react-scripts start",
     "storybook": "start-storybook -p 9009 -s public",
     "test": "react-scripts test",
-    "upload": "python scheme/data.py && NODE_ENV=development node scheme/uploadMock.js"
+    "upload": "NODE_ENV=development node scheme/uploadMock.js",
+    "dev": "REACT_APP_USE_DB_EMULATORS=true npm-run-all --sequential dev:functions_build  --parallel dev:firebase dev:react dev:upload",
+    "dev:functions_build": "cd functions && npm run build && cd ..",
+    "dev:firebase": "firebase emulators:start",
+    "dev:react": "react-scripts start",
```

Comment:
Is this a duplicate for `start`?

Reply:
yeah, but it is subject to change in case of environment, so its better with duplicated